react/week1/todo-list/src/index.js

```diff
@@ -1,18 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
+import Todo from './App';
+import todos from './todos';
+
 
-const Todo = function (props) {
-  return <li>{props.todo}, {props.deadline}</li>
-}
 ReactDOM.render(
   <>
     <h1>Todo List</h1>
     <ul>
-      <Todo todo="Get out of bed" deadline={"Wed Sep 13 2017"} />
-      <Todo todo="Brush teeth" deadline={"Thu Sep 14 2017"} />
-      <Todo todo="Eat breakfast" deadline={"Fri Sep 15 2017"} />
+      {todos.map((aTodo) => <Todo todo={aTodo.todo} deadline={aTodo.deadline} />)}
```

Comment:
Hi Simon
Good job with the homework.
I have small things  to mention:
Name your components (file)
To figure which component has a bug, it’s important to always give your component a name.  I would name App.js to be 
Todo.js

The console is showing a warning that says Each child in a list should have a unique "key" prop.

you can avoid the problem by providing a unique key that has the value of todo.id 
<Todo todo={aTodo.todo} deadline={aTodo.deadline}  key={aTodo.id}/>
 It is also good to have [ Naming Conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md) to name your variables

Reply:
sure .. thanks